packages/firestore/test/unit/util/bundle_data.ts

```diff
@@ -73,14 +78,28 @@ export class TestBundleBuilder {
     });
     return this;
   }
+
   addNamedQuery(
     name: string,
     readTime: api.Timestamp,
-    bundledQuery: BundledQuery
+    query: Query,
+    limitType?: LimitType
```

Comment:
Query already has the LimitType.

Reply:
Would it at all be possible to do the Query translation (flip the order) here? Not sure if feasible, but then we could drop the LimitType and reduce the confusion.

Reply:
Done.